code/parachain/frame/pallet-multihop-xcm-ibc/src/lib.rs

```diff
@@ -435,7 +504,7 @@ pub mod pallet {
 				});
```

Comment:
Github doesn’t let me comment on the actual line, but

```rust
route.sort_by(|a, b| a.0.order.cmp(&b.0.order));
```

a few lines above can be written as:

```rust
route.sort_by_key(|item| &item.0);
```